### PR TITLE
lesbar: init at 1.1.0

### PR DESCRIPTION
**Pull request by jpentland**
## Description of changes

Added new package lesbar
https://git.sr.ht/~salmiak/lesbar

## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) (or backporting [23.05 Release notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md))
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by jpentland** [comment]
@prusnak

File: pkgs/applications/window-managers/lesbar/default.nix [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  lib,
+  stdenv,
+  fetchFromSourcehut,
+  pkgs,
+}:
+stdenv.mkDerivation (finalAttrs: let
+  version = "1.1.0";
+in {
+  inherit version;
+  pname = "lesbar";
```
Comment (PassiveLemon) [inline pkgs/applications/window-managers/lesbar/default.nix:11]: I think you should use this instead:
```
stdenv.mkDerivation rec {
  pname = "lesbar";
  version = "1.10";
  ...
```
rec allows you to use attributes defined in the same scope

**Comment by prusnak** [comment]
@GrahamcOfBorg build lesbar

File: pkgs/applications/window-managers/lesbar/default.nix (hunk 1 above)
Comment (prusnak) [reply pkgs/applications/window-managers/lesbar/default.nix:12]: apparently `finalAttrs` is now preferred

Comment (PassiveLemon) [reply pkgs/applications/window-managers/lesbar/default.nix:12]: Interesting...

**Comment by prusnak** [comment]
Merged. Thanks for the contribution!

**Pull request merged**